src/libs/Navigation/AppNavigator/AuthScreens.tsx

```diff
@@ -403,6 +407,32 @@ function AuthScreens({session, lastOpenedPublicRoomID, initialLastUpdateIDApplie
         // eslint-disable-next-line react-compiler/react-compiler, react-hooks/exhaustive-deps
     }, []);
 
+    const clearStatus = () => {
+        User.clearCustomStatus();
+        User.updateDraftCustomStatus({
+            text: '',
+            emojiCode: '',
+            clearAfter: DateUtils.getEndOfToday(),
+        });
+    };
+    useEffect(() => {
+        const currentTime = format(new Date(), 'yyyy-MM-dd HH:mm:ss');
+        if (!currentUserPersonalDetails.status?.clearAfter) {
+            return;
+        }
+        if (currentUserPersonalDetails.status?.clearAfter > currentTime) {
```

Comment:
I think it's safer to compare milliseconds than string though the date format should be consistent.